tests/download.bats

```diff
@@ -4,21 +4,31 @@ load "${BATS_PLUGIN_PATH}/load.bash"
 
 load '../lib/download.bash'
 
+is_stubbed=false
```

Comment:
Mutable global variables are a bit of a pest. Consider whether this is really necessary any more. If it is, add a comment. If it isn't (my suspicion), remove it here and all references.